README.md

```diff
@@ -61,7 +61,7 @@ before we can include support for Debian 8 and Debian, respectively._
 Operating System | MySQL | PostgreSQL
 ---------------- | ------| -----------
 CentOS 7         | 5.6   |   9.3
-Debian 7         | 5.5   |   9.3
+Debian 7         | 5.5   |  10.1
```

Comment:
Do you get 10.1 on Debian 7? This page indicates that it's 9.1: https://packages.debian.org/search?searchon=names&keywords=postgresql

That also matches what I get when I try to install postgresql on Debian 7 / Wheezy.

Reply:
The added source repository gives version 10.1.